Bottom: 2ab9c75fa5a7a5e489a379dff428cbf574782892
Top:    2d6afd34e7492ed723b76c198230f5258288a301
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 19:37:47 +0000

Refresh of temp-polygon-mess

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 9157c50..888903d 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,7 +39,6 @@
       are marked
 */
 
-//#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
